### DISCUSSION TITLE
Single auth setup for all downstream swaggers

### DISCUSSION THREAD

**Discussion by andreykalachev** [Burgyn/MMLib.SwaggerForOcelot#169] [Q&A]
Hi,

I set up API Key authentication for my Ocelot gateway. So my APIs don't have any authentication only the gateway does. However, to access them through swagger I still need to authenticate.
Is it possible to set up swagger authentication in the getaway and reuse it across all downstream APIs?

**Comment by Burgyn** [comment]
Hi @andreykalachev,

please read this issue: https://github.com/Burgyn/MMLib.SwaggerForOcelot/issues/83
